arroyo-api/src/rest.rs

```diff
@@ -77,17 +79,25 @@ pub fn create_rest_app(server: ApiServer, pool: Pool) -> Router {
 
     // TODO: enable in development only!!!
     let cors = CorsLayer::new()
+        .allow_methods(cors::Any)
         .allow_headers(cors::Any)
         .allow_origin(cors::Any);
 
+    let jobs_routes = Router::new()
+        .route("/", get(get_pipeline_jobs))
+        .route("/:job_id/errors", get(get_job_errors))
+        .route("/:job_id/checkpoints", get(get_job_checkpoints));
+
     let api_routes = Router::new()
         .route("/ping", get(ping))
+        .route("/validate", post(validate_pipeline))
```

Comment:
what's your though on having this top-level vs. nested under pipelines?

Reply:
Done